packages/terra-dev-site/src/webpack/plugin/SitePlugin.js

```diff
@@ -236,18 +236,23 @@ class SitePlugin {
     }).apply(compiler);
   }
 
-  apply(compiler) {
-    const isWebpack5 = compiler.webpack && compiler.webpack.version.startsWith('5');
+  static getPublicPath({ compiler }) {
+    if (process.env.TERRA_DEV_SITE_PUBLIC_PATH) {
+      compiler.options.output.publicPath = process.env.TERRA_DEV_SITE_PUBLIC_PATH;
+      return process.env.TERRA_DEV_SITE_PUBLIC_PATH;
+    }
 
-    // Use default public path else the env else /
-    let defaultPublicPath;
-    if (compiler.options.output && compiler.options.output.publicPath) {
-      defaultPublicPath = compiler.options.output.publicPath;
+    if (compiler.options.output && compiler.options.output.publicPath && compiler.options.output.publicPath !== 'auto') {
+      return compiler.options.output.publicPath;
     }
-    const publicPath = process.env.TERRA_DEV_SITE_PUBLIC_PATH || defaultPublicPath || '/';
+    compiler.options.output.publicPath = '/';
+    return '/';
```

Comment:
Can we just return `compiler.options.output.publicPath`.

Reply:
in this case we either don't have a default or it's set to auto so we want to explicitly set it to / for our routing to work.